client/src/main/java/org/threadly/litesockets/client/http/HTTPClient.java

```diff
@@ -587,5 +595,22 @@ public MergedByteBuffers getBody() {
     public String getBodyAsString() {
       return body.duplicate().getAsString(body.remaining());
     }
+    
+    public Element getBodyAsXml() throws SAXException {
```

Comment:
Why should XML get special treatment?  I think people will be more likely to want to use a parsing library of their choice anyways.